<div>Note : 1) Please insert those corresponding ref by DOI.</div><div>&nbsp; 2) Check previous writings and integrate them to here such as previous manuscripts , seed funding proposal writing, poster, transfer proposal<table><tbody><tr><td></td></tr></tbody></table></div><h1>1 Introduction</h1><div>Whole exome sequencing has been widely used for the identification of disease-associated genetic variants. For severe and rare Mendelian diseases, it has been proven to be an effective approach (Agha, et al., 2014). Exome sequencing may also contribute to the understanding of the aetiology of complex diseases, e.g., Schizophrenia (Purcell, et al., 2014) and Type 2 Diabetes (Lohmueller, et al., 2013). However, the functional annotation of the protein-coding genes is still far from complete. By November 2014, 19,007 protein-coding genes had been registered in the HUGO Gene Nomenclature Committee (HGNC) database (Gray, et al., 2013) which includes most of the relevant DNA regions responsible for severe Mendelian diseases and some complex diseases. On the other hand, the data from the 1000 Genome Project indicates that an apparently healthy person could carry approximately 250~300 loss-of-function (LoF) single nucleotide variants (SNVs) on average (2010; Via Garc√≠a and Genomes Project, 2012). Therefore, the accurate estimation of gene and variant pathogenicity and corresponding etiologic architecture pose a considerable challenge to human genetic community.</div><div>A number of bioinformatics tools based on predicted deleteriousness and distinct biological features have been developed to prioritize protein-changing (i.e., non-synonymous) variants in the human protein-coding genes. The tools FATHMM <cite class="ltx_cite raw v1">\cite{Shihab_2012}</cite>, GERP <cite class="ltx_cite raw v1">\cite{Cooper_2005}</cite>, LRT <cite class="ltx_cite raw v1">\cite{Chun_2009}</cite>, MutationAssessor <cite class="ltx_cite raw v1">\cite{Reva_2011}</cite>, SIFT <cite class="ltx_cite raw v1">\cite{Ng_2003}</cite>, SiPhy <cite class="ltx_cite raw v1">\cite{Garber_2009}</cite> and PROVEAN <cite class="ltx_cite raw v1">\cite{Choi_2012}</cite> mainly consider sequence similarities and conservation patterns across various species, whereas CADD <cite class="ltx_cite raw v1">\cite{Kircher_2014}</cite>, MutationTaster <cite class="ltx_cite raw v1">\cite{Schwarz_2014}</cite> and Polyphen2 <cite class="ltx_cite raw v1">\cite{Adzhubei_2010}</cite> consider the functional information of the genetic variations from multiple different information sources in their algorithms. Moreover, the Mutation Significance Cutoff (MSC) uses gene-level dynamic thresholds generated by either CADD, PolyPhen-2 or SIFT to reduce false negative rate (Yuval, et al., 2016). These tools are particularly useful in clinical sequencing studies of Mendelian diseases as to exclude a large number of neutral variants. However, inconsistencies in prediction results among all these different variant-based prediction tools have been demonstrated (L and results from variant-based functional prediction are hard to be generalized into the gene-level.</div><div>A few gene scoring systems have been proposed to estimate the effect of genetic variations at the gene-level. Firstly, Haploinsufficiency score (HI) (Huang, et al., 2010) combines a list of biological properties by inspecting the copy number variations (CNV) among thousands of healthy individuals. In this system, genes with a higher HI score may imply their potential of causing dominant traits. Secondly, Recessive score (REC) (MacArthur, et al., 2012) uses 213 known LoF tolerant genes and 858 known recessive disease genes to build up a linear discriminant model to estimate the probability that a gene can cause a recessive disease. Thirdly, Genic Intolerance score (RVIS) (Petrovski, et al., 2013) assesses functional genetic variation tolerance ability from profiles of SNVs with allele frequency information. Fourthly, network indispensability score (NET) (Khurana, et al., 2013) calculates the gene centrality and indispensability in various protein-protein interactions and regulatory networks to dissect the gene importance. Fifthly, Gene Constraint score (CONS) (Samocha, et al., 2014) compares the genomic background mutation profile with the observed mutation profile of a gene for interpreting <i>de novo</i> mutations. Lastly, human gene damage index (GDI) (Itan, et al., 2015) also utilizes the mutation profile in both monogenic disease patients and the general population to prioritize exome variants. However, each of these popular gene scoring systems can only explain part of the genetic architecture from different perspectives. For example, i)HI estimation does not include the profiles of non-CNV genetic variants ; ii)the REC score does not consider the dominant disease-predisposing genes; iii) the RVIS score does not deal with variations of allele frequencies in different populations; iv) the systematic comparison for different known disease associated genes is missing in NET score study and the sample size of LoF-tolerant and essential genes are moderated; v) the CONS score is only applicable for the interpretation of <i>de novo</i> mutations; and vi) the GDI score only considers mutation profiles. Therefore, more sophisticated analysis and comprehensive prioritization schemes on genes causing human disease are needed.</div><div>One of the most important characteristics of a Mendelian disease is that its mode of inheritance can be inferred from the pedigree. However, only a few studies have partially examined the unique biological features of genes causing the different inheritance modes of Mendelian diseases (Huang, et al., 2010; MacArthur, et al., 2012). Therefore, we developed a new gene-level scoring system, Inheritance Modes Specific Pathogenicity Prioritization (ISPP) for the prediction of inheritance-specific pathogenicity. In addition to the consideration of the different inheritance modes, ISPP also combines six existing gene-level prioritization systems with many gene features. The performance of the proposed approach was evaluated systematically by multiple curated disease-associated gene sets.</div><h1>2 Methods</h1><h3>Collection of gene features and benchmark datasets</h3><div></div><div><u>Variant level, gene level and functional related gene features:</u></div><div>Population variant profiles in protein-coding genes were obtained from the dataset of NHLBI GO Exome Sequencing Project (ESP6500), which includes data from more than 200,000 individuals from over 20 clinical sequencing studies. We extracted the variant profiles of all protein-coding genes by using KGGSeq (Li, et al., 2012) and counted the number of non-synonymous (Nonsyn_v) and synonymous (Syn_v) variants in each gene with approved gene symbol from HUGO Gene Nomenclature Committee (HGNC). Moreover, we retrieved 18 biological gene features from the Ensembl database (Flicek, et al., 2014) through the BioMart system. These features include gene length (Length), GC content (GC), total number of RNA isoforms (ALL_tr_count), mean length of all RNA isoforms (ALL_tr_length), mean length of the protein coding RNA isoforms (CO_tr_length), number of protein coding (CO_tr_count) and non-coding (NC_tr_count) RNA isoforms (Kasprzyk, 2011). In addition, we also gathered functional-related gene features (Lawrence, et al., 2013), including i) global expression (Expr) derived from RNA-Seq data and summed across the tens different cell lines, ii) DNA replication time (Reptime), iii) the Hi-C statistic (HIc), a measurement of chromatin statuses as well as iv) the noncoding mutation rate (NC_mut_rate) which measured the mutation rate of intronic regions. Moreover, published gene prioritization scores, HI (Huang, et al., 2010), REC (MacArthur, et al., 2012), RVIS (Petrovski, et al., 2013), NET (Khurana, et al., 2013), CONS (Samocha, et al., 2014) and GDI (Itan, et al., 2015) were downloaded from either dbNSFP (Liu, et al., 2013) or their corresponding studies. Tissue-specific expression information (Tissuespe) was obtained from Liu et.al (Liu, et al., 2008). See S1 Table for details.</div><div></div><div><u>Benchmark disease-associated gene sets:</u></div><div>Based on the OMIM database (URL:<a href="http://www.omim.org/">http://www.omim.org/</a> Online Mendelian Inheritance in Man,), ten lists of experts-curated disease genes were aggregated as hOMIM gene lists (Blekhman, et al., 2008). They are hOMIM_AD (419 autosomal dominant disease genes), hOMIM_AR (569 autosomal recessive disease genes), hOMIM_AD_AR (39 both autosomal dominant and recessive disease genes), hOMIM_XL (66 X-linked disease genes), hOMIM_Birth (637 genes for birth-onset diseases ), hOMIM_Pre15 (9 genes associated with diseases with onset age before 15), hOMIM_Pre-rep (368 genes associated with diseases with the onset age after 15 and before 40), hOMIM_Post40 (33 genes associated with diseases with onset age after 40), hOMIM_Cancer (548 cancer genes from the COSMIC database), hOMIM_essential (hOMIM_ess) (1,643 genes lethal if knocked-out in mice). We merged hOMIM_Birth and hOMIM_Pre15 <img></div><div>Likewise, similar but extended gene lists for diseases with different inheritance modes and onset ages were obtained from the Clinical Genomic Database (CGD) (Solomon, et al., 2013): CGD_AD (876 autosomal dominant disease genes), CGD_AR (1,500 autosomal recessive disease genes), CGD_AD_AR (304 genes with both autosomal dominant and recessive inheritance modes), CGD_XL (182 X-linked Mendelian disease genes), CGD_Adult (69 genes) and CGD_Paediatric (1,401 genes). We also collected the genes for both complex and Mendelian disorders characterized by Jin W, et.al (2012) (Jin, et al., 2012). MC (Mendelian and Complex diseases, 525 genes), MNC (Mendelian but Not Complex diseases, 445 genes) as well as CNM (Complex but Not Mendelian diseases, 2,594 genes). See S1 Table for details.</div><div></div><h3>Permutation test for exploring pattern specific model</h3><div>In order to assess the biological and functional features of each disease-associated gene set, we computed the mean of each feature for each set and compared them with randomly generated gene sets. For each disease-associated gene set, we 1) computed the observed mean (<i>X<sub>i</sub></i>) of each feature <i>i</i>; 2) obtained the sampling distribution under null hypothesis and the sampling mean <i>¬µ<sub>i</sub></i><sub> </sub>&nbsp;and variance <i>œÉ<sub>i</sub></i> of each feature <i>i</i> from selecting 100,000 random gene sets with the same gene number as the observed gene set; 3) calculated a z-score for feature <i>i</i> of the disease-associated gene set, <i>Z</i><sub>i</sub>, based on the central limit theorem, i.e., <i>Z<sub>i</sub></i> = (<i>X<sub>i</sub></i> - <i>¬µ<sub>i</sub></i>)/<i>œÉ<sub>i</sub></i> (Figure 2, S2 Table); and 4) calculated the corresponding p-value of the z-score. A Bonferroni-corrected p-value threshold of 0.05 / 520 ‚âà 9.615 √ó 10E-05 was used to declare significance.</div><div></div><h3>Machine learning model construction from selected gene features</h3><div>For differentiating gene sets with different inheritance modes and age of onset, we used the random forest algorithm which was implemented in Weka package (<a href="http://www.cs.waikato.ac.nz/ml/weka/">http://www.cs.waikato.ac.nz/ml/weka/</a>) to construct the machine learning models with different combinations of gene features. By using CGD database as the training dataset, we constructed all possible combinations of biological features to test the performance of the model. We compared the critical predictors when different feature subsets can achieve similar performance. The 10-fold cross-validation which also included non-training genes was used to assess the performance of the models. The best model was defined as the one with the largest Area Under Curve (AUC) of Receiver Operating Characteristics (ROC). In the ISPP_XL model, the cross validation which were only sampled from those genes on X chromosome.</div><div></div><h1>3 Results</h1><div>In this study, we first collected 30 known disease-associated gene lists from independent studies or public databases as the benchmark dataset for building and testing the inheritance-specific disease gene prediction models. While various, these disease-associated gene lists are not necessarily mutually exclusive. We included nine gene groups of different inheritance modes and eight gene groups for disease with different ages of onset. Besides, we collected 14 gene features as well as six widely used gene prioritization scores as predictors in our prediction models (See Methods for details). The data collection and analysis framework are summarized in Supplementary S1 Table and Figure 1.</div><div></div><h2>3.1 Existing gene prioritization scores unsatisfactory distinguish among the disease-associated gene lists</h2><div>We first compared the performance of six popular gene-based prioritization methods (HI, REC, RVIS, NET, CONS and GDI) which assess gene pathogenicity using different perspectives across the 26 disease-associated gene lists. Spearman Correlation analysis suggests that these six methods only have modest correlation with each other (rexcept for the correlation between HI and REC (r=0.77). (S1 Figure, S2 Table and S1 Text)</div><div>We found that the REC score had the best performance in predicting disease-predisposing genes among all six gene prioritization scores examined (Figure 2). However, the REC scores were insensitive to inheritance modes even though the program was designed to carry out the recessive disease causation probability. For instance, both hOMIM_AD (Z score=23.9) and hOMIM_AR (Z score=21.8) had significantly high REC scores compared to the random gene sets in permutation test; also, hOMIM_AD even had a marginally higher REC score (recessive disease causation probability) than hOMIM_AR (Rank sum p-value = 0.061) (S2 Figure). Similarly, CGD_AD had significantly higher REC scores than CGD_AR (Wilcoxon Rank-Sum p-value = 1.554E-07).</div><div>For the HI score, the performance in classifying autosomal recessive genes from random gene sets was poor. All recessive disease-associated genes in the lists RVIS_AR, hOMIM_AR and CGD_AR, as well as Non_ess genes had no difference or significantly lower HI score compared to random gene sets (S3 Table, S2 Text). Also, in HI scores, the AUC of CGD_AD and CGD_AR training gene sets were only, respectively, 0.69 and 0.49 (Figure 3). The NET scores estimated the indispensability of each gene by interaction network with moderate performance on disease-associated genes. The AUC of NET scores can reach 0.7 and 0.62 on CGD_AD and CGD_AR respectively.</div><div>The purpose of the RVIS score is to estimate the tolerance of a gene with functional variations; as a result, most of the disease-associated gene lists had significantly lower RVIS scores than random, especially hOMIM_AD (p = 5.91E-14), hOMIM_early (p = 1.11E-10) and RVIS_Denovo (p = 4.68E-26). However, the RVIS scores failed to separate recessive disease-associated genes (RVIS_AR, hOMIM_AR and CGD_AR), and the AUC only had, respectively, 0.64 and 0.54 in CGD_AD and CGD_AR (Figure 3). The similar idea was used in GDI score to build a gene damage index from the mutation profile of those known diagnosed monogenic disease subjects. However, the AUC of GDI score only had 0.45 and 0.58 in CGD_AD and CGD_AR respectively.</div><div><img></div><h2>3.2 Each disease-associated gene list had its own unique pattern of biological and functional characteristics</h2><div>We observed that most disease gene lists examined consistently had significantly more non-synonymous variants, synonymous variants and a longer gene length than random in the permutation test (Figure 2, S3 Table). However, there was no clear variant profile for classifying disease genes according to just the different inheritance modes. The only exception was the X-linked disease-associated genes. Both hOMIM_XL and CGD_XL with average gene length (p = 5.9E-02, 5.6E-04, respectively; Bonferroni-corrected p-value ‚âà 9.615E-05) had significantly fewer non-synonymous variants (p = 2.79E-10, 1.36E-10, respectively) and synonymous variants (p = 3.72E-25, 1.09E-25, respectively). We also found that recessive disease-associated genes (hOMIM_AR, RVIS_AR and CGD_AR) had significantly much more RNA non-coding isoforms (p ‚â§ 2.70E-8). Moreover, for hOMIM, RVIS and CGD studies, all those recessive disease-associated genes had more non-synonymous variants than that corresponding to dominant disease-associated genes (p ‚â§ 5.69E-40) (S3, S4 Text). Besides, genes in hOMIM_AR and CGD_AR had significantly smaller CONS scores than random gene sets (p = 9.82E-8 and 1.24E-15, respectively), suggesting that the recessive disease-associated genes may be less sensitive to <i>de novo</i> mutations. Surprisingly, there was no statistical difference for CONS score among CGD_Paediatric, MC and hOMIM_early, suggesting that <i>de novo</i> mutations may have a benign effect on these early-onset disease-associated genes. Furthermore, CGD_AD, Genic_Denovo, hOMIM_Cancer and hOMIM_essential had higher Z scores for HI and CONS scores than CGD_Paediatric and hOMIM_early, implying that dominant disease inheritance mechanisms are more likely to be due to haploinsufficiency or <i>de novo</i> mutation sensitive.</div><div><img></div><div><b>&nbsp;</b></div><div><b>&nbsp;</b></div><div><b>Figure 3. Performance comparisons of different gene prioritization methods.</b> ROC curves of seven gene prioritization approaches‚Äô capacity to predict the corresponding independent disease-associated gene lists. HI=Haploinsufficiency score, REC=Recessive score, RVIS= Genic Intolerance score, NET= Network indispensability score, CONS=Gene constraint score, GDI=Gene Damage Index and ISPP=10 fold cross-validation of corresponding machine learning model.</div><div></div><h2>3.3 Development of a disease inheritance mode specific gene prioritization system, incorporating the unique patterns of biological and functional characteristics</h2><div>Here, we have built a unified framework by using the random forest approach to combine multiple biological features and prioritization scores for predicting pathogenic genes with specific inheritance modes. The training datasets included CGD_AD (876 genes), CGD_AR (1,500 genes), CGD_XL (182 genes). Ten-fold cross-validation was adopted to compare the performance of the prediction models, which was quantified by the AUC (See Methods). For dominant diseases, our combined model can achieve slightly better performance in CGD_AD (with an AUC of 0.75) compared to the individual prediction systems. Surprisingly, the REC scores performed much better than HI scores in separating dominant disease genes from random genes with AUC of 0.73 and 0.69 respectively (Figure 3). For recessive diseases, our combined model had much better performance than other prediction algorithms in CGD_AR with an AUC of 0.73. Moreover, the performance of our model was considerably superior to that of the other methods regarding the prioritization of X-linked disease-associated genes with an AUC of 0.85 in CGD_XL. However, the AUC of our combined model was similar to the AUC values obtained by REC scores in hOMIM_early and CGD_Paediatric (S3 Figure). Finally, we have annotated 18859 protein-coding genes and provided a set of pathogenic scores on each gene (S4 Table) for dominant, recessive and X-linked (ISPP_AD, ISPP_AR and ISPP_XL scores).</div><h2>3.4 <img></h2><div>We validated the performance of our combined prediction model by a series of independent gene sets. Venn diagrams (S4A Figure) show the degree of overlap among various training data (CGD_AD, CGD_AR and CGD_XL) and other reported recessive disease-associated gene lists (hOMIM_AR, REC recessive and RVIS_Recessive). There were respectively 128, 107, and 206 recessive disease genes which were reported independently but not be included in our training sets. We further compared the discrepancies of their ISPP_AD and ISPP_AR scores by Wilcoxon rank sum test. In these three recessive gene sub-sets, the ISPP_AR scores from the combined model are significantly higher than the of ISPP_AD scores for all of the non-overlapping sub-sets (p ‚â§ 2.206E-5) (Figure S4B).</div><div>For each inheritance mode, we demonstrated the ISPP scores for 5 novel disease-causal genes which recently published by independent next-generation sequencing studies. None of these genes were included in our positive training gene sets. As shown in Table 1, all of the genes have the highest pathogenic prediction scores in the corresponding inheritance modes and the scores are consistent with the mode of inheritance initially described per each gene. Additionally, we validated the results in one of the recent largest X-exome sequencing study on intellectual disability disease (Hu, et al., 2015). The study reported seven novel and validated XLID (X-linked Intellectual Disability) genes (<i>CLCN4, CNKSR2, FRMPD4, KLHL15, LAS1L, RLIM</i> and <i>USP27X</i>) as well as two novel candidate XLID genes (<i>CDK16</i> and <i>TAF1</i>). Seven of nine genes are on the top 32% of the ISPP_XL prediction model in ISPP_XL score system. <i>TAF1</i> is in the CGD_XL training set and has highest score on ISPP_XL score among other inheritance modes (S4 Table). Even though <i>RLIM</i> and <i>USP27X</i> have relatively low predicted ISPP_XL score, they still have highest predicted score among other ISPP scores. This suggests that the consideration of inheritance mode can enhance the pathogenicity prediction.</div><div>&nbsp;</div><div>&nbsp;</div><div><b>Table 1. Examples of published candidate genes with mode of inheritance and corresponding ISPP scores.</b></div><div>&nbsp;</div><div><table><tbody><tr><td><div><b>Autosomal Dominant</b></div></td></tr><tr><td><div>Gene</div></td><td><div>ISPP_AD</div></td><td><div>ISPP_AR</div></td><td><div>ISPP_XL</div></td><td><div>Ref</div></td></tr><tr><td><div><i>APP</i></div></td><td><div>0.17(5%)</div></td><td><div>0.01(69%)</div></td><td><div>-</div></td><td><div>(Conidi, et al., 2015)</div></td></tr><tr><td><div><i>IRS1</i></div></td><td><div>0.12(6%)</div></td><td><div>0.04(29%)</div></td><td><div>-</div></td><td><div>(Rong, et al., 2015)</div></td></tr><tr><td><div><i>RET</i></div></td><td><div>0.17(5%)</div></td><td><div>0.01(69%)</div></td><td><div>-</div></td><td><div>(Figlioli, et al., 2012)</div></td></tr><tr><td><div><i>NCOR1</i></div></td><td><div>0.23(5%)</div></td><td><div>0.01(80%)</div></td><td><div>-</div></td><td><div>(Fozzatti, et al., 2011)</div></td></tr><tr><td><div><i>EGR2</i></div></td><td><div>0.20(5%)</div></td><td><div>0.01(76%)</div></td><td><div>-</div></td><td><div>(Lupski, et al., 2010)</div></td></tr><tr><td><div><b>Autosomal Recessive</b></div></td></tr><tr><td><div>Gene</div></td><td><div>ISPP_AD</div></td><td><div>ISPP_AR</div></td><td><div>ISPP_XL</div></td><td><div>Ref</div></td></tr><tr><td><div><i>KMT2B</i></div></td><td><div>0.01(48%)</div></td><td><div>0.03(43%)</div></td><td><div>-</div></td><td><div>(Agha, et al., 2014)</div></td></tr><tr><td><div><i>APOB</i></div></td><td><div>0.11(6%)</div></td><td><div>0.25(8%)</div></td><td><div>-</div></td><td><div>(Hammer, et al., 2013)</div></td></tr><tr><td><div><i>GJB2</i></div></td><td><div>0.02(37%)</div></td><td><div>0.07(17%)</div></td><td><div>-</div></td><td><div>(Nikolay, et al., 2011)</div></td></tr><tr><td><div><i>FECH</i></div></td><td><div>0.01(46%)</div></td><td><div>0.18(8%)</div></td><td><div>-</div></td><td><div>(Balwani, et al., 2013)</div></td></tr><tr><td><div><i>ATF6</i></div></td><td><div>0.02(28%)</div></td><td><div>0.1(12%)</div></td><td><div>-</div></td><td><div>(Ansar, et al., 2015)</div></td></tr><tr><td><div><b>X-Linked</b></div></td></tr><tr><td><div>Gene</div></td><td><div>ISPP_AD</div></td><td><div>ISPP_AR</div></td><td><div>ISPP_XL</div></td><td><div>Ref</div></td></tr><tr><td><div><i>POLA1</i></div></td><td><div>0.05(13%)</div></td><td><div>0.01(74%)</div></td><td><div>0.34(19%)</div></td><td><div>NIH</div></td></tr><tr><td><div><i>OGT</i></div></td><td><div>0.02(34%)</div></td><td><div>0.00(80%)</div></td><td><div>0.28(21%)</div></td><td><div>(Niranjan, et al., 2015)</div></td></tr><tr><td><div><i>GLUD2</i></div></td><td><div>0.01(57%)</div></td><td><div>0.00(80%)</div></td><td><div>0.18(24%)</div></td><td><div>(Cukier, et al., 2014)</div></td></tr><tr><td><div><i>BRCC3</i></div></td><td><div>0.00(63%)</div></td><td><div>0.02(61%)</div></td><td><div>0.16(26%)</div></td><td><div>(Huang, et al., 2015)</div></td></tr><tr><td><div><i>ZMYM3</i></div></td><td><div>0.00(63%)</div></td><td><div>0.00(80%)</div></td><td><div>0.25(22%)</div></td><td><div>(Philips, et al., 2014)</div></td></tr><tr><td><div><i>CLCN4</i></div></td><td><div>0.02(32%)</div></td><td><div>0.00(80%)</div></td><td><div>0.21(22%)</div></td><td><div>(Hu, et al., 2015)</div></td></tr><tr><td><div><i>CNKSR2</i></div></td><td><div>0.00(63%)</div></td><td><div>0.00(80%)</div></td><td><div>0.12(32%)</div></td><td><div>(Hu, et al., 2015)</div></td></tr><tr><td><div><i>FRMPD4</i></div></td><td><div>0.00(63%)</div></td><td><div>0.04(34%)</div></td><td><div>0.19(24%)</div></td><td><div>(Hu, et al., 2015)</div></td></tr><tr><td><div><i>KLHL15</i></div></td><td><div>0.03(27%)</div></td><td><div>0.00(80%)</div></td><td><div>0.19(24%)</div></td><td><div>(Hu, et al., 2015)</div></td></tr><tr><td><div><i>LAS1L</i></div></td><td><div>0.02(34%)</div></td><td><div>0.01(66%)</div></td><td><div>0.13(30%)</div></td><td><div>(Hu, et al., 2015)</div></td></tr><tr><td><div><i>RLIM</i></div></td><td><div>0.00(63%)</div></td><td><div>0.00(80%)</div></td><td><div>0.05(47%)</div></td><td><div>(Hu, et al., 2015)</div></td></tr><tr><td><div><i>USP27X</i></div></td><td><div>0.00(63%)</div></td><td><div>0.00(80%)</div></td><td><div>0.03(57%)</div></td><td><div>(Hu, et al., 2015)</div></td></tr><tr><td><div><i>CDK16</i></div></td><td><div>0.00(63%)</div></td><td><div>0.01(74%)</div></td><td><div>0.15(28%)</div></td><td><div>(Hu, et al., 2015)</div></td></tr></tbody></table></div><div>By comparing ISPP scores of each gene, the possible inheritance mode of the gene could be estimated (S6 Text). The value in parentheses indicates the percentage among all in the inheritance-specific model. NIH indicates this x-linked disease candidate gene has not been published yet, but has been funded for conducting the research by NIH. Project#:1R56AI113274-01</div><div></div><h1>4 Conclusions</h1><div>By examining a range of well-studied and independent disease-associated gene lists, our results suggest that most of the existing methods cannot satisfactorily distinguish gene pathogenicity and inheritance mode together among all the various gene lists. In the present study, we investigated the characteristics of Mendelian disease genes with different inheritance modes. We found that autosomal dominant disease-associated genes tend to be <i>de novo</i> mutation sensitive and are prone to haploinsufficiency whereas autosomal recessive disease-associated genes are likely to have more non-synonymous variants and non-coding RNA isoforms. Moreover, the X-linked disease-associated genes have significantly fewer non-synonymous and synonymous variants. This framework is splendid to combine RNA level information with current gene prioritization methods for predicting pathogenic protein coding gene under different inheritance modes.</div><div>The correlation analysis indicated that the overall expression level, Hi-C status and GC content had a positive correlation with each other (S3 Text). Moreover, almost all of the disease-associated genes had much more RNA isoforms than randomly selected gene sets (S4 Text). Even though the information bias might be one of the confounding factors, some disease-associated gene lists are not as significant as others such as hOMIM_AD and CGD_XL. Additionally, the recessive disease-associated genes, in particular, had more complicated regulatory processes and genetic variant tolerant ability, and the early-onset disease-associated genes may have <i>de novo</i> mutations with merely neutral effect. Apart from GDI scores, none of the gene prioritization scores examined had a strong correlation with the gene features we investigated (S1, S3 Text). This suggests that the gene features we included have no predominant effect on the combined machine learning prediction model. Interestingly, we found that the REC score can well predict all of the disease-associated gene lists we collected with significantly higher recessive disease causation probability (REC, p ‚â§ 1.99E-8). However, it seemed no power for REC to discriminate dominant and recessive disease genes.</div><div>Figure 2 shows the dendrogram of cluster analysis based on the empirical Z scores for comparing gene feature patterns among 26 disease-associated gene lists (S5 Text). All recessive disease-associated gene lists from three independent studies were clustered together, but there are only 165 overlapping genes in these gene lists. All of these three lists had no difference or significantly lower CONS, RVIS and HI scores (Figure 2 and S3 Table). On the other hand, based on the cluster analysis, dominant disease-associated gene lists (including CGD_AD, RVIS_Denovo, hOMIM_Cancer and hOMIM_essential gene sets) had significantly higher Z scores for various transcriptional features, and HI and CONS scores. Besides, it‚Äôs also fair and complementary to use other prediction methods genome-wide regardless the mode of inheritance. Each prediction has its own proposed usage which might not be fully illustrated by ISPP scores only. In conclusion, our inheritance mode-specific prediction model was able to assess the potential pathogenicity of a gene accurately. It also reflects the various biological characteristics behind genes for diseases with different inheritance modes. These information could provide valuable information for gene annotation.</div><div>could provide valuable information for gene annotation in the future.</div><div>&nbsp;</div>