<div>Note : 1) Please insert those corresponding ref by DOI.&nbsp;</div><div>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;2) Check previou</div><div></div><div>1 Introduction</div><div></div><div>Whole exome sequencing has been
used for identifying disease-associated genetic variations. For severely rare
Mendelian diseases, it has been proven to be an effective approach (Agha, et
al., 2014). Ex-ome sequencing may also contribute to the understanding of the
aetiol-ogy of complex diseases. For example, a meta-analysis of genome-wide
association studies of schizophrenia by the Psychiatric Genomics Consortium
identified 108 disease-associated genetic loci and the re-sults can be compared
and contrasted with the exome sequencing find-ings (Schizophrenia Working Group
of the Psychiatric Genomics, 2014). However, the functional annotation of the
protein-coding genes is still far from complete. By November 2014, 19,007
protein-coding genes have been registered in HUGO Gene Nomenclature Committee
(HGNC) database (Gray, et al., 2013), which include the most of the relevant
DNA regions responsible for severe Mendelian diseases and some complex
diseases. On the other hand, the data from the 1000 Genome Project indicates
that an apparently healthy person could carry approximately 250~300
loss-of-function (LoF) single nucleotide vari-ants (SNVs) on average (2010; Via
García and Genomes Project, 2012). Therefore, the estimation of gene
pathogenicity and correspond-ing etiologic architecture for diseases pose a
considerable challenge to human genetic community.</div><div> As
a result, a number of bioinformatics tools have been developed to prioritize
protein-changing (i.e., non-synonymous) vari-ants in the human protein-coding genes
based on predicted deleterious-ness and distinct biological features. The tools
FATHMM (Shihab, et al., 2013), GERP (Cooper, et al., 2005), LRT (Sung and Fay,
2009), MutationAssessor (Reva, et al., 2011), SIFT (Ng and Henikoff, 2003),
SiPhy (Garber, et al., 2009) and PROVEAN (Choi, et al., 2012) mainly consider
sequence similarities and conservation patterns across various species, whereas
CADD (Kircher, et al., 2014), MutationTaster (Jana Marie, et al., 2014) and
Polyphen2 (Ivan, et al., 2010) consider the functional information of the
genetic variations from multiple different information sources in their
algorithms. These tools are particularly useful in clinical sequencing studies
of Mendelian diseases as the large number of candidate variants found often
precludes the functional follow-ups or replications of all these candidates.
However, inconsist-encies in prediction results among all these different
variant-based prediction tools have been demonstrated (Li, et al., 2012) and
results from variant-based functional prediction are usually hard to be
gener-alized to the gene-level. Therefore, the variant-level prediction alone
may not be sufficient, and the interpretation of the genuine
disease-predisposing genetic variants from neutral remains ambiguous.</div><div>A few gene scoring systems have
been proposed to estimate the effect of genetic variations at the gene-level.
The first is the Haploinsuffi-ciency score (HI) (Huang, et al., 2010) which
combined a list of bio-logical properties by inspecting the copy number
variations (CNV) among thousands of healthy individuals. In this system, genes
with a higher HI score may imply their relationships with dominant traits (and
vice versa). The second one is the Recessive score (REC) (MacArthur, et al.,
2012) which used 213 known LoF tolerant genes and 858 known recessive disease
genes to build up a linear discriminant model to esti-mate the probability that
a mutation causes a recessive disease. The third one is the Genic Intolerance
score (GENIC) (Petrovski, et al., 2013) which assessed functional genetic
variation tolerance ability from profiles of SNVs with allele frequency
information. The last one is the Gene Constraint score (CONS) (Samocha, et al.,
2014) which was computed by comparing the genomic background mutation profile
with the observed mutation profile of a gene for interpreting de novo
mutations. However, each of these popular gene scoring systems has its own
disadvantages. For example, the profiles of non-CNV genetic variants were not
taken into account in HI estimation; the REC score does not consider dominant
disease genes; the GENIC score does not deal with variations of allele
frequencies in different populations; and the CONS score is only developed for
the interpretation of de novo mutations. Therefore, more sophisticated analyses
and prioritization schemes would be needed.</div><div>One of the most important
characteristics of a Mendelian disease is its inheritance mode in pedigrees.
However, only a few studies have par-tially examined the unique biological features
of genes causing Mende-lian diseases with different modes of inheritance
(Huang, et al., 2010; MacArthur, et al., 2012). Therefore, using a systematic
machine learn-ing approach, we developed a gene-level scoring system which
takes the disease inheritance modes into account together with other gene
prioritization scores and a number of genic features for accurately predicting
the ability of a gene to cause Mendelian diseases of a specif-ic inheritance
mode compared to other genes in the human genome.</div><div></div><div>2 Methods</div><div></div><div>Collection of gene features and
benchmark datasets</div><div>Variant level, gene level and
functional related gene features: </div><div>Population variant profiles in
protein-coding genes were obtained from the dataset of NHLBI GO Exome
Sequencing Project (ESP6500), which included data from more than 200,000
individuals collected from &gt; 20 clinical sequencing studies. We extracted
the genetic variants’ profile in protein-coding genes from the dataset by
KGGseq (Li, et al., 2012). Variant profiles also included allele frequency
information of non-synonymous and synonymous variants in 1000 Genomes Project.
We calculated the number of non-synonymous and synonymous vari-ants in each
gene approved by the HUGO Gene Nomenclature Com-mittee (HGNC). Moreover, we
retrieved 18 other biological gene fea-tures from the Ensembl database (Flicek,
et al., 2014) (including gene length, GC content, RNA isoform numbers, mean
length of all RNA isoforms, mean length of protein coding RNA isoforms, protein
coding and non-coding RNA isoform numbers) through the BioMart system
(Kasprzyk, 2011). We also gathered some functional-related gene features
related to the background gene mutation rate (Lawrence, et al., 2013),
including I) global expression, derived from RNA-Seq data and summed across the
tens different cell lines, ii) DNA replication time, iii) the Hi-C statistic, a
measurement of chromatin statuses as well as iv) the noncoding mutation rate
which measures the rate of mutation in the intronic region of a gene. Finally,
gene prioritization scores, HI (Huang, et al., 2010), REC (MacArthur, et al.,
2012), GENIC (Petrovski, et al., 2013) and CONS (Samocha, et al., 2014) were
down-loaded from dbNSFP (Liu, et al., 2013) and the corresponding studies.
Tissue-specific expression information was obtained from Liu et.al (Liu, et
al., 2008). See S1 Table for details.</div><div>Benchmark disease-associated gene
sets: </div><div>Ten lists of experts-curated
disease genes were aggregated based on the OMIM database (Online Mendelian
Inheritance in Man) as hO-MIM gene lists (Blekhman, et al., 2008). They were hOMIM_AD (consisting of 419
autosomal dominant disease genes), hOMIM_AR (consisting of 569 autosomal
recessive disease genes), hO-MIM_AD_AR (consisting of 39 autosomal dominant and
recessive disease genes), hOMIM_XL (consisting of 66 X-linked disease genes),
hOMIM_Birth (consisting of 637 genes for diseases onset on birth), hOMIM_Pre15
(consisting of 9 genes for diseases with an onset age before 15), hOMIM_Pre-rep
(consisting of 368 genes for diseases with an onset age after 15 and before
40), hOMIM_Post40 (consisting of 33 genes for diseases with an onset age after
40), hOMIM_Cancer (con-sisting of 548
cancer genes from the COSMIC database), hO-MIM_essential (consisting of 1,643
genes which were lethal if being knocked-out in mice). We merged hOMIM_Birth
and hOMIM_Pre15 into hOMIM_earlyOnset (consisting of 646 genes for diseases
with an onset age before 15 or birth) since hOMIM_Pre15 only had 9 genes. In
addition, we collected gene lists for different disease contexts from the GENIC
study, including Genic_Haploinsufficiency (175 genes), Domi-nant-negative (364
genes), Genic_Recessive (817 genes), Gen-ic_Denovo (467 genes), and
Genic_Denovo plus Haploinsufficiency (109 genes). Two other gene lists from the
MGI database (Blake, et al., 2014), MGI_Lethality (92 genes) and MGI_Seizure
(95 genes) genes, were also selected.</div><div>Furthermore, we constructed two
non-essential gene lists by analyz-ing the dbNSFP dataset. The non-essential
gene (Non_ess) must have at least one non-synonymous variant: 1) with a minor
allele frequency (MAF) q &gt; 3.3% (816 genes), assuming that the frequency of
homozy-gous recessive in the population is q2 &gt; 0.1%; as well as 2)
predicted to be as deleterious by at least four aforementioned variant-based func-tional
prediction algorithms.</div><div>Likewise, similar but extended gene
lists for diseases with different inheritance modes and onset ages were
obtained from the Clinical Genomic Database (CGD) (Solomon, et al., 2013):
CGD_AD (consist-ing of 876 autosomal dominant disease genes), CGD_AR
(consisting of 1,500 autosomal recessive disease genes), CGD_AD_AR (consisting
of 304 genes with both autosomal dominant and recessive mode of inher-itance),
CGD_XL (consisting of 182 X-linked Mendelian disease genes), CGD_Adult
(consisting of 69 genes ) and CGD_Paediatric (consisting of 1,401 genes). </div><div>In order to have a more
comprehensive annotation of the protein-coding genes, we also collected other
gene lists from the public data-base and previous literature. Also, we obtained
the genes for both com-plex and Mendelian disorders characterized by Jin W,
et.al (2011) (Jin, et al., 2012). MC (consisting of 525 genes responsible for
both Mende-lian and complex diseases), MNC (consisting of 445 genes responsible
for Mendelian but not complex diseases) as well as CNM (consisting of 2,594
genes responsible for complex but not Mendelian diseases).</div><div>Permutation test for exploring
pattern specific model</div><div>Also, in order to assess the
difference in the patterns of biological and functional features between
disease-susceptibility genes and ge-nome background, for each
disease-associated gene set, we 1) com-puted the observed mean (Xi) of each
feature i; 2)got a null sampling distribution (with a sampling mean µi, and a
sampling variance σi2) of the mean of feature i from 100,000 random gene sets
of the same size; 3)calculated a z-score for feature i for the
disease-associated gene set, Zi, based on the central limit theorem, i.e., Zi =
(Xi - µi)/σi (Figure 2, S2 Table); and 4)obtained the corresponding p-value of
the z-score calculated. A Bonferroni-corrected p-value threshold of 0.05 / 468
≈ 1.068 × 10-4 was used to declare significance. </div><div>Constructing machine learning model
from gene features</div><div>We used a random forest implemented
in Weka (<a href="http://www.cs.waika">http://www.cs.waika</a> <a href="http://to.ac.nz/ml/weka/">to.ac.nz/ml/weka/</a>) to construct a machine learning
model with different combinations of features and prioritization scores as
predictors of the various genes with different inheritance modes and age of
onset. By using the gene lists from the CGD database as the training dataset,
we constructed the power sets of all biological features to test the
perfor-mance of the model. We compared the critical predictors when several
different feature subsets can achieve similar performance and differ-ent disease
gene lists were used as the training dataset. In the XL mod-el, the random
genes which were only sampled from those on X chro-mosome.10-fold
cross-validation was used to assess the performance of the models for various
modes of inheritance constructed using the sig-nificant predictors. The best
model was defined as the one with the largest Area Under Curve (AUC). </div><div></div><div>3 Results</div><div></div><div>In this study, we first collected
30 known disease-associated gene lists from previous independent studies and
public databases as the bench-mark training datasets for building and testing
the inheritance-specific disease gene prediction models. While various, these
disease-associated gene lists are not necessarily mutually exclusive. We
includ-ed nine gene groups of different inheritance modes and eight gene groups
for disease with different ages of onset. Besides, we collected 14 gene
features as well as 4 widely-used gene prioritization scores as predictors in
our prediction models (See Methods for details). The data collection and
analysis framework are summarized in Supplementary S1 Table and Figure 1. </div><div>&nbsp;</div><div>Figure 1: Schematic diagram
illustrating the construction of the inheritance modes specific gene
prioritization model.</div><div></div><div>3.1 Existing
gene prioritization scores distinguished among the different disease-associated
gene lists unsatisfactorily</div><div>We first compared the performance
of the four popular gene-based prioritization methods, for distinguishing among
the different disease-associated gene lists. Correlation analysis suggests that
only HI and REC had the strongest Spearman correlation (S1 Figure, S2 Table),
indicating that these four methods only have modest overlaps with each other
(S1 Text) </div><div>Surprisingly, we found that the REC
score had unexpectedly good per-formance in predicting disease-associated genes
among all four gene prioritization scores examined. However, Figure 2 and S3
Table show that the REC scores failed to carry out both the correct inheritance
modes and recessive disease causation probability. For instance, both hOMIM_AD
(Z score=23.9) and hOMIM_AR (Z score=21.8) had significantly high REC scores,
hOMIM_AD even had a marginally higher REC score (recessive disease causation
probability) when com-pared with hOMIM_AR (Rank sum p-value = 0.061) (S2
Figure). The same was observed when we compared the REC score in CGD_AD with
that in CGD_AR (Rank sum p-value = 1.554E-07). </div><div>For the HI scores, the performance
in classifying autosomal recessive genes from random gene sets was poor. All
recessive disease-associated genes in the lists Genic_Recessive, hOMIM_AR and
CGD_AR, as well as Non_ess genes had no difference or significantly lower HI
score compared to random genes (S3 Table, S2 Text). Also, the AUC of HI score
only had, respectively, 0.69 and 0.49 in CGD_AD and CGD_AR (Figure 3). </div><div>The GENIC score was purposed to
estimate the tolerant ability of a gene with functional variations, and most
disease-associated gene lists had significantly lower GENIC score, especially
hOMIM_AD (p = 5.91E-14), hOMIM_earlyOnset (p = 1.11E-10) and Genic_Denovo (p =
4.68E-26). However, the GENIC scores failed to separate recessive
disease-associated genes (Genic_Recessive, hOMIM_AR and CGD_AR) from random
genes, and the AUC only had, respectively, 0.64 and 0.54 in CGD_AD and CGD_AR
(Figure 3).</div><div>The CONS score was originally aimed
at estimating the de novo muta-tion effect of a gene. By comparing the genomic
background mutation rate, higher score suggests higher effect of de novo
mutations. The gene lists hOMIM_AD and CGD_AD inclined to have significantly
higher CONS score (p = 2.62E-10 and p = 4.97E-51, respectively), implying that
these genes are more vulnerable to de novo mutations (S2 Text), and this is
consistent with the definition of dominant diseases. However, the AUC of CONS
score only had, respectively, 0.65 and 0.43 in CGD_AD and CGD_AR (Figure 3). </div><div>&nbsp;</div><div>Figure 2. Gene characteristics
versus disease-associated gene lists heat map (with clus-tering based on the
empirical significance levels). Red colors represent higher value than expected
from random gene sets of the same size. The attached dendrogram on the left
depicts hierarchical clustering results for the disease-associated gene lists.
We can identify lists of genes associated with diseases with similar mode of
inheritance clustered together by exhibiting similar gene feature patterns. See
Materials and Methods for details.</div><div>&nbsp;</div><div>&nbsp;Figure 3. Performance comparisons
of different gene prioritization methods. ROC curves of five gene
prioritization approaches’ capacity to predict the corresponding inde-pendent
disease-associated gene lists. Cons=constraint score, Genic= Genic Intolerance
score, Rece=Recessive score, HI=Haploinsufficiency score and 10x=10 fold
cross-validation of corresponding machine learning model.</div><div>3.2 Each
disease-associated gene list had its own unique pattern of biological and
functional characteristics</div><div></div><div>We observed that most disease gene
lists examined consistently had significantly more non-synonymous variants,
synonymous variants and a longer gene length (Figure 2, S3 Table). However,
there was no clear variant profile for classifying disease genes with various
inheritance modes. The only exception was the X-linked disease-associated
genes. Both hOMIM_XL and CGD_XL had normal gene length with signifi-cantly
fewer non-synonymous variant (p = 2.79E-10, 1.36E-10, respec-tively) and
synonymous variants (p = 3.72E-25, 1.09E-25, respective-ly). We also found that
two recessive disease-associated gene lists (hOMIM_AR and CGD_AR) had
significantly much more RNA regu-latory isoforms (p ≤ 2.70E-8) and
non-synonymous variants (p ≤ 5.69E-40) than CGD_AD (S3, S4 Text). Besides, genes
in hOMIM_AR and CGD_AR had significantly smaller CONS scores (p = 9.82E-8 and
1.24E-15, respectively), suggesting that the recessive disease-associated genes
may have less effect on de novo mutations. Moreover, surprisingly, there was no
statistical difference for CONS score among CGD_Paediatric, MC and
hOMIM_earlyOnset, suggesting that early-onset disease-associated genes may have
de novo mutations with mere-ly neutral effect. Furthermore, CGD_AD,
Genic_Denovo, hO-MIM_Cancer and hOMIM_essential had higher Z scores for HI and
CONS scores than early-onset disease genes like CGD_Paediatric and
hOMIM_earlyOnset, implying that dominant disease inheritance mech-anisms may be
more haploinsufficient and de novo mutation sensitive.</div><div>3.3 Development
of a disease inheritance mode specific gene prioritization system,
incorporating the unique patterns of biological and functional characteristics </div><div>Here, we built a unified framework
by the random forest approach to combine multiple biological features and
prioritization scores for pre-dicting pathogenic genes with specific
inheritance modes. The training datasets included CGD_AD (876 genes), CGD_AR
(1,500 genes), CGD_XL (182 genes). 10-fold cross-validation was adopted to
com-pare the performance of the prediction models, which was quantified by the
AUC (See Methods). For dominant diseases, compared to the other prediction
scores, our combined model can achieve similar and even slightly better
performance in CGD_AD (with an AUC of 0.75). Surprisingly, the REC scores had much
better performance in separat-ing dominant disease genes from random genes
(with an AUC of 0.73) than the HI scores (with an AUC of 0.69) in CGD_AD
(Figure 3). For recessive diseases, our combined model has much better
performance in CGD_AR with an AUC of 0.72. Moreover, our combined model had a
substantial improvement in performance for prioritizing X-linked
disease-associated genes, with an AUC of 0.84 in CGD_XL. However, our combined
model only had similar AUC compared to the REC scores in hOMIM_earlyOnset and
CGD_Paediatric (S3 Figure). In final, we annotated 18859 protein-coding genes
and provided a set of pathogenic scores on each gene (S4 Table) for dominant,
recessive and X-linked (AD, AR and XL scores). </div><div></div><div>3.4 Validation
of the new gene prioritization system</div><div>&nbsp;</div><div>To validate the performance of our
combined prediction model, we checked the prediction scores for genes which
were not included in the training dataset. Venn diagrams (S4A Figure) show the
degrees of overlapping among various training data (CGD_AD, CGD_AR and CGD_XL)
and other recessive disease-associated gene lists (hO-MIM_AR, REC recessive and
Genic_Recessive). There were 128, 107, and 206 genes respectively not in the AR
training set. We compared the discrepancies of their AD and AR scores using a
rank sum test. S4B Figure shows the AR scores from the combined model are
significantly higher than the AD scores for all of the non-overlapping gene
sets (p ≤ 2.206E-5).</div><div>For each inheritance mode, we
selected 5 novel disease-causal genes recently identified by next-generation
sequencing studies. None of these genes were included in our training datasets.
As shown in Table 1, all of the genes have the highest pathogenic prediction
scores in the inheritance modes which are consistent with the disease studies
in the original sequencing studies. Additionally, we validated the results in
one of the largest X-exome sequencing study currently (Hu, et al., 2015). The
study reported seven novel and validated XLID (X-linked Intellec-tual
Disability) genes (CLCN4, CNKSR2, FRMPD4, KLHL15, LAS1L, RLIM and USP27X) as
well as two novel candidate XLID genes (CDK16 and TAF1). Seven of nine genes
are on the top 45.1% of the combined prediction model, and all genes have at
least 0.02 (Top 61.2%) in XL combined prediction score system. </div><div>&nbsp;</div><div>Table 1. Example of genes
associated with diseases known to have a specific mode of inheritance from
published literatures and their corresponding prediction scores.&nbsp;</div><div>By comparing a set of score of a
gene for inheritance mode specific models, the possible inheritance mode of the
gene could be estimated. The value in paren-theses indicates the percentage
among all in the inheritance-specific model. NIH indicates this x-linked
disease candidate gene has not been published yet, but has been funded for
conducting the research by NIH. Project#:1R56AI113274-01</div><div></div><div>4 Conclusions</div><div>&nbsp;</div><div>Our results suggested that all
existing gene prioritization scores could not fully distinguish among all the
various disease-associated gene lists satisfactorily. A range of well-studied
and independent disease-associated gene lists were examined. In the present
study, we investi-gated the characteristics for Mendelian disease genes with
different inheritance modes. We found
that autosomal dominant disease-associated genes tend to harbor de novo
mutation and be more haploin-sufficient, whereas autosomal recessive disease
associated genes are more likely to have non-synonymous variants and RNA
isoforms. Moreover, the X-linked disease-associated genes have significantly
fewer both non-synonymous and synonymous variants. Therefore, this framework is
splendid to combine transcription level information to current gene
prioritization systems for predicting pathogenic human protein coding gene
under different inheritance modes.</div><div>Correlation analysis indicated that
the overall expression level, Hi-C status and GC content had a positive
correlation with each other, and the DNA replication timing had a negative
correlation with the length of the RNA isoforms (S3 Text). Moreover, almost all
of the disease-associated genes had much more RNA isoforms than randomly
selected gene sets of the same size (S4 Text) Additionally, the recessive
dis-ease-associated genes, in particular, had more complicated regulatory
processes and genetic variant tolerant ability, and the early-onset
dis-ease-associated genes may have de novo mutations with merely neutral
effect. </div><div>Interestingly, even though both HI
and REC scores make use of various gene features in their computation, none of
the gene prioritization scores examined had a strong correlation with the gene
features we investigated (S1, S3 Text). This suggests that the gene features we
included have no predominant effect on the combined machine learn-ing
prediction model. Furthermore, we found that the REC score can well predict
many disease-causal genes as all disease-associated gene lists we collected had
significantly higher recessive disease causation probability (REC, p ≤
1.99E-8). But it is interesting to note that not all of the disease-associated
gene lists came from diseases with a recessive inheritance mode. </div><div>Figure 2 shows the dendrogram of
cluster analysis based on the empiri-cal Z scores for comparing gene feature
patterns among 12 disease-associated gene lists (S5 Text). In order to reduce
the noise, only gene lists with more than 450 genes (except for CGD_XL) were
included. All recessive disease-associated gene lists from independent studies
were clustered together but there are only 165 overlapping genes in these gene
lists. All of these three lists had no difference or significant-ly lower CONS,
GENIC and HI scores (Figure 2 and S3 Table). On the other hand, based on the
cluster analysis, dominant disease-associated gene lists (including CGD_AD,
Genic_Denovo, hOMIM_Cancer and hOMIM_essential gene sets) had significantly
higher Z scores for vari-ous transcriptional features, and HI and CONS scores.
In conclusion, our inheritance mode-specific prediction model was able to
assess the potential pathogenicity of a gene accurately. It did not only enable
the prediction of the corresponding independent disease-associated gene lists,
but also reflect the various biological characteristics behind genes for
diseases with different inheritance modes and onset ages. These information
could provide valuable information for gene annotation in the future.</div><div>&nbsp;</div>